README.md

```diff
@@ -117,7 +117,7 @@ Then, add the locally built library as a dependency in your application's build.
 ```
 dependencies {
 ...
-    implementation ("com.splunk:splunk-otel-android:0.14.0-SNAPSHOT")
+    implementation ("com.splunk:splunk-otel-android:0.14.0")
```

Comment:
This is about the local build, it's still going to be `0.14.0-SNAPSHOT`

Reply:
Changed back. On last release this was changed in pre and back in post, I guess that was unintended there then.